servatrice/src/servatrice.cpp

```diff
@@ -405,6 +405,8 @@ bool Servatrice::initServer()
         }
     }
 #endif
+
+    qDebug() << "Idle client timeout value: " << getIdleClientTimeout();
```

Comment:
The motivating scenario:

You're wooger. You run a server. You open your logs and see this line in them:
`DEBUG Idle client timeout value: 50`

What do you do?